purge-build-queue-by-name.groovy

```diff
@@ -1,5 +1,11 @@
 //Purge build queue by name
 
+// find tasks by name which contains string pattern REPLACEME
 import hudson.model.*
```

Comment:
```suggestion

```
This one is implicit so no need to explicitly import. Also `Jenkins` class is in the `jenkins.model` package, not in `hudson.model` so pointless anyway.